single_tech_samples/databricks/sample1_basic_azure_databricks_environment/README.md

```diff
@@ -0,0 +1,210 @@
+
+Below listed are the steps to deploy this sample :
+
+1. Fork and clone this repository. Navigate to (CD) `single_tech_samples/databricks/sample1_basic_azure_databricks_environment/`.
+
+1. The sample depends on the following environment variables to be set before the deployment script is run:
+  
+    > - `DEPLOYMENT_PREFIX` - Prefix for the resource names which will be created as a part of this deployment
+    > - `AZURE_SUBSCRIPTION_ID` - Subscription ID of the Azure subscription where the resources should be deployed.
+    > - `AZURE_RESOURCE_GROUP_NAME` - Name of the containing resource group
+    > - `AZURE_RESOURCE_GROUP_LOCATION` - Azure region where the resources will be deployed. (e.g. australiaeast, eastus, etc.)
+    > - `DELETE_RESOURCE_GROUP` - Flag to indicate the cleanup step for the resource group
+
+1. Run '/deploy.sh'
+   > Note: The script will prompt you to log in to the Azure account for authorization to deploy resources.
+
+   The script will validate the ARM templates and the environment variables before deploying the resources. It will also display the status of each stage of the deployment while it executes. The following screenshot displays the log for a successful run:
+
+   > Note: `DEPLOYMENT_PREFIX` for this deployment was set as `lumustest`
+
+    ![alt text](../Common_Assets/Images/IAC_Script_Deploy.png "Logo Title Text 1")
+
+### 3.3. Deployed Resources
+
+The following resources will be deployed as a part of this sample once the script is executed:
+
+1.Azure Databricks workspace.
+
+![alt text](../Common_Assets/Images/IAC_Adb.png "Logo Title Text 1")
+
+2.Azure Storage with hierarchical namespace enabled.
+
+![alt text](../Common_Assets/Images/IAC_Storage.png "Logo Title Text 1")
+
+3.Azure Key vault with all the secrets configured.
+
+![alt text](../Common_Assets/Images/IAC_Keyvault.png "Logo Title Text 1")
+
+### 3.4. Deployment validation
+
+The following steps can be performed to validate the correct deployment of this sample:
+
+1. Users with appropriate access rights should be able to:
+
+   1. launch the workspace from the Azure portal.
+   2. Access the control plane for the storage account and key vault through the Azure portal.
+   3. View the secrets configured in the Azure Key vault
+   4. View deployment logs in the Azure resource group
+   ![alt text](../Common_Assets/Images/IAC_Deployment_Logs.png "Logo Title Text 1")
+
+### 3.5. Clean-up
+
+Please follow the below steps to clean up your environment :
+
+The clean-up script can be executed to clean up the resources provisioned in this sample. Following are the steps to execute the script:
+
+1. Navigate to (CD) `single_tech_samples/databricks/sample1_basic_azure_databricks_environment/`.
+
+2. Run '/destroy.sh'
+
+The following screenshot displays the log for a successful clean-up run:
+
+  ![alt text](../Common_Assets/Images/IAC_Script_Teardown.png "Logo Title Text 1")
```

Comment:
Add Next Steps section pointing to sample 2.

Reply:
Addessed in coming commits.

Reply:
@namitms Addressing